app/controllers/admin/base_controller.rb

```diff
@@ -3,6 +3,7 @@ class Admin::BaseController < ActionController::Base
   before_action :authenticate_user!
   protect_from_forgery with: :exception
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+  after_action :verify_authorized
```

Comment:
this is from Punditi document say if i forget add authorizaton to action this method remind me on that when not add before